RecoJets/JetProducers/plugins/CSJetProducer.cc

```diff
@@ -143,10 +168,20 @@ void CSJetProducer::fillDescriptions(edm::ConfigurationDescriptions& description
 void CSJetProducer::fillDescriptionsFromCSJetProducer(edm::ParameterSetDescription& desc) {
   desc.add<double>("csRParam", -1.);
   desc.add<double>("csAlpha", 2.);
-
+  desc.add<bool>("useModulatedRho", false);
+  desc.add<double>("minFlowChi2Prob", false);
+  desc.add<double>("maxFlowChi2Prob", false);
   desc.add<edm::InputTag>("etaMap", edm::InputTag("hiFJRhoProducer", "mapEtaEdges"));
   desc.add<edm::InputTag>("rho", edm::InputTag("hiFJRhoProducer", "mapToRho"));
   desc.add<edm::InputTag>("rhom", edm::InputTag("hiFJRhoProducer", "mapToRhoM"));
+  desc.add<edm::InputTag>("rhoFlowFitParams", edm::InputTag("hiFJRhoFlowModulationProducer", "rhoFlowFitParams"));
```

Comment:
```suggestion
  desc.add<edm::InputTag>("rhoFlowFitParams", {"hiFJRhoFlowModulationProducer", "rhoFlowFitParams"});
```
a bit shorter; but update the above as well, if apply this one